js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -2088,10 +2088,12 @@ function processQuery(query, explain, planIndex) {
           filter = '   ' + keyword('FILTER') + ' ' + buildExpression(info.filter);
         }
         let accessString = '';
-        if (!info.indexCoversProjections) {
+        if (!info.indexCoversProjections && info.producesOutput) {
           accessString += "index scan + document lookup";
-        } else {
+        } else if (info.indexCoversProjections) {
           accessString += "index scan";
+        } else {
+          accessString += "index scan only";
```

Comment:
This confuses me. The difference between "index scan" and "index scan only" is not clear to me :D

Reply:
well, it index scan (and copy values out of the index) vs only scan through the index and don't care about the values you find...